### PR TITLE
[UXIT-1482] Security Maturity Model Page Header

### PR DESCRIPTION
**Pull request by mirhamasala**
## 📝 Description

This PR adds the Security Maturity Model page with a page header. 

- **Type:** New feature

## 🛠️ Key Changes
- Adds `security/maturity-model/page.tsx`

## 📌 To-Do Before Merging

- [x] Check if we have the page header image yet [>>]

## 🧪 How to Test

Go to `/security/maturity-model` and make sure the page works

## 📸 Screenshots

CleanShot 2024-08-29 at 13 37 30@2x

### CODE REVIEW

File: public/admin/config.yml [hunk 1]
```diff
@@ -245,6 +245,13 @@ collections:
             max: 6
           - *meta_config
 
+      - name: "maturity-model"
+        label: "Maturity Model"
+        file: "src/content/pages/maturity-model.md"
```
Comment (barbaraperic) [inline public/admin/config.yml:250]: missing preview. I'll open a PR to fix the template

File: src/content/pages/maturity-model.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+header:
+  title: "Web3 Maturity Model for Security"
```
Comment (barbaraperic) [inline src/content/pages/maturity-model.md:3]: think the text changed? https://www.figma.com/design/0pBfQs5tKI6bx5ypPIfVRA/FIL.org-V4-Prototypes?node-id=5796-72478&t=yktIYZU22ebTzIqm-4

File: public/admin/config.yml [hunk 3]
```diff
@@ -245,6 +245,13 @@ collections:
             max: 6
           - *meta_config
 
+      - name: "maturity-model"
```
Comment (barbaraperic) [inline public/admin/config.yml:248]:
```suggestion
      - name: "security/maturity-model"
```

File: public/admin/config.yml (hunk 1 above)
Comment (barbaraperic) [inline public/admin/config.yml:250]:
```suggestion
        file: "src/content/pages/security/maturity-model.md"
```

File: cypress/e2e/critical/maturity_model_page_spec.cy.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { PATHS } from '@/constants/paths'
+import { testPageMetadata } from '@/support/test-utils'
+import { verifyLinks } from '@/support/verifyLinksUtil'
+
+describe('Maturity Model Page', () => {
+  it('should check metadata', () => {
+    testPageMetadata(PATHS.MATURITY_MODEL)
```
Comment (barbaraperic) [inline cypress/e2e/critical/maturity_model_page_spec.cy.ts:7]: `overrideDefaultTitle: true` ?

Comment (mirhamasala) [reply cypress/e2e/critical/maturity_model_page_spec.cy.ts:7]: It will depend on SEO recommendations..

**Pull request merged**